### ISSUE TITLE
Choose colors of chips

### ISSUE THREAD

**Issue by rileyjohngibbs** [rileyjohngibbs/ConnectFour_CASComSci_2019-2020#3]
A player should be able to change the colors of the board and of each player's chips, in order to improve color blindness accessibility and to match the different iterations of the game that used different colors.
